src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -28,6 +29,15 @@ export const getChangeRequestByID = async (req: Request, res: Response) => {
   return res.status(200).json(changeRequestTransformer(requestedCR));
 };
 
+//returns whether the given description bullet it checked
+export const checkForUncheckedDescriptionBullets = (
```

Comment:
this should be in the desc bullets utils file

also whats the point of this if you're not even using hte list and just checking one value of the input?